uavcan/node/62805.Heartbeat.1.0.uavcan

```diff
@@ -0,0 +1,47 @@
+#
+# Abstract node status information.
+#
+# All UAVCAN nodes are required to publish this message periodically.
+# This is the only data type that must be supported by all nodes.
+#
+# The default subject ID 62805 is 1111010101010101 in binary. The alternating bit pattern at the end
+# helps transceiver synchronization (e.g., on CAN-based networks) and on some transports permits
+# automatic bit rate detection (e.g., CAN bus).
+#
+
+# Publication period must not exceed this limit.
+# The period should not change as long as the node is running.
+uint16 MAX_PUBLICATION_PERIOD = 1   # [second]
+
+# If a node fails to publish this message in this amount of time, it should be considered offline.
+uint16 OFFLINE_TIMEOUT = 3          # [second]
+
+# The uptime seconds counter should never overflow. The counter will reach the upper limit in ~136 years,
+# upon which time it should stay at 0xFFFFFFFF until the node is restarted.
+#
+# Other nodes may detect that a remote node has restarted when this value jumps backwards.
+uint32 uptime                       # [second]
+
+# Abstract node health.
+uint2 HEALTH_STATUS_OK       = 0    # Nominal. Color: none or green.
+uint2 HEALTH_STATUS_ADVISORY = 1    # Minor parameter deviation or minor issue. Color: blue/cyan.
+uint2 HEALTH_STATUS_CAUTION  = 2    # Major problem or failure. Color: amber/yellow.
+uint2 HEALTH_STATUS_WARNING  = 3    # Fault, loss of function. Color: red.
+truncated uint2 health_status
```

Comment:
I like this change a lot. But I feel there is still weirdness coming from the mix of "health" with the aviation words. When I read the documents I feel these words are intended to be grouped with "light", "staus" or "indicator". Is there any reason `health_status` was used over just `status` or `status_indication`